src/components/legend/draw.js

```diff
@@ -326,38 +325,29 @@ module.exports = function draw(gd) {
     }
 
     if(gd._context.editable) {
-        var xf,
-            yf,
-            x0,
-            y0,
-            lw,
-            lh;
+        var xf, yf, x0, y0;
```

Comment:
@mdtusz if you feel adventurous, maybe you add our first `editable: true` jasmine test ...